pkg/limayaml/validate.go

```diff
@@ -136,6 +136,12 @@ func Validate(y *LimaYAML, warn bool) error {
 		return fmt.Errorf("field `memory` has an invalid value: %w", err)
 	}
 
+	for i, disk := range y.AdditionalDisks {
+		if disk.Name == "" {
+			return fmt.Errorf("additionalDisks[%d]: disk name is required", i)
```

Comment:
To be consistent with error messages in this function:
```suggestion
			return fmt.Errorf("field `additionalDisks[%d]` must be provided", i)
```